yazi-core/src/tab/selected.rs

```diff
@@ -36,7 +40,7 @@ impl Selected {
 			*self.parents.entry(u).or_insert(0) += urls.len();
 		}
 
-		self.inner.extend(urls.iter().cloned().cloned());
+		self.inner.extend(urls.iter().map(|&u| u.clone()));
```

Comment:
I've tried many ways to get this to compile here, thank you very much for your approach, it's been very enlightening for me.